### ISSUE TITLE
Disable swiping in opposite direction once started in one direction

### ISSUE THREAD

**Issue by tomlokhorst** [MortimerGoro/MGSwipeTableCell#179]
Feature Request:

It would be nice if swiping to left is disabled, once a swipe to right has started. 
The only way to swipe to left would be to let go of the screen and start again.

This is similar to the way Swipe cells work on iOS 9 Mail app.

**Comment by MortimerGoro** [comment]
I have added a property called allow allowsOppositeSwipe in the new 1.5.4 release. You can set NO value to achieve the effect you want.

**Comment by tomlokhorst** [comment]
This works great! Thanks a bunch!

A nice-to-have future improvement could be to have the cell "bounce" a bit in the opposite direction. Like the Mail app.

When allowsOppositeSwipe=false, the buttons in the opposite direction are hidden, and the user can swipe "a bit" in the opposite direction. I'm not entirely sure how Mail does the "a bit" part, I think it decreases the elasticity of the swiping.

**Comment by MortimerGoro** [comment]
@tomlokhorst Yes, I wanted to add the the bounce but it requires more work. I will try to implement it in the next release ;)
